index.html

```diff
@@ -137,11 +137,11 @@ <h3>Looks like your system can run Freckle, if you have trouble please send us a
       <footer class="row">
         <div class="col-xs-12 text-center uv-padding">
           <p>
-            <a target="_blank" href="https://freckle.zendesk.com">More information about Freckle</a>
+            <a target="_blank" href="https://support.freckle.com">More information about Freckle</a>
           </p>
           <p>
             Still need help?
-            <a target="_blank" href="https://freckle.zendesk.com/hc/en-us/requests/new">Contact Support</a>
+            <a target="_blank" href="https://support.freckle.com/hc/en-us/requests/new">Contact Support</a>
```

Comment:
Actually, I didn't realize we were asking people to file tickets on Zendesk, which we don't use for that purpose, only for knowledge base. Can we use https://frontrowed.wufoo.com/forms/freckle-support-request/ as a link instead?

It's an external domain AND it uses frontrowed (... I know...) but at the very least we are monitoring that inbox. Long term the plan is to get off of Wufoo, we just haven't executed on that yet.